components/ui-api-layer/docs/resource-quotas-status.md

```diff
@@ -0,0 +1,116 @@
+# ResourceQuotasStatus 
+
+[ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) provides constraints that limit aggregate resource consumption per Namespace.
+
+The ResourceQuota values can be as follows:
+```bash
+Name:            kyma-default
+Resource         Used       Hard
+--------         ---        ---
+limits.memory    7348440Ki  10Gi
+requests.memory  3727576Ki  7Gi
+```
+The `used` values of the ResourceQuota can exceed the `hard` values and block any resource creation in the Environment.
```

Comment:
``When the `used` values of the ResourceQuota exceed the `hard` values, the resource creation in the Environment is blocked.``